src/components/Pill/__tests__/Pill.spec.js

```diff
@@ -1,21 +1,51 @@
 /**
  * @file Pill tests.
- * @copyright IBM Security 2019
+ * @copyright IBM Security 2020
  */
 
-import { shallow } from 'enzyme';
+import { render } from '@testing-library/react';
 import React from 'react';
+import userEvent from '@testing-library/user-event';
 
-import Pill from '../';
+import { Pill } from '../../..';
 
-describe('Pill', () => {
-  let pill;
+describe('PanelV2', () => {
+  test('should have no Axe or DAP violations', async () => {
+    const main = document.createElement('main');
+    render(<Pill value="127.0.0.1" type="IP" />, {
+      // DAP requires a landmark '<main>' in the DOM:
+      container: document.body.appendChild(main),
+    });
+    await expect(document.body).toHaveNoAxeViolations();
+    await expect(document.body).toHaveNoDAPViolations('Pill');
+  });
+
+  test('should add a custom class', () => {
```

Comment:
👏Good! I believe we'll also need tests that verify the `inline`, `type`, and `value` props

Reply:
🤔 `inline` & `type` don't change... seems more appropriate to check in the `Decorator` anyway?
Also, `value` is included in every text but I can make a specific one...

Reply:
Actually about `value`,  think of it this way:

I'm querying `getByText(/127.0.0.1/i)` multiple times. It wouldn't work if the `value` prop wasn't working as expected. 👍 In this way, checking what the user sees is already demonstrating the functionality/internals of the component.

Reply:
The reason I'm calling these out is that they're props that can be provided and configured by consumers, but I feel like this again goes back to this grey area where we're still defining our boundaries when passing props down to a child component! 

If a consumer applies the `inline` prop at the `Pill` level, I believe that needs to be verified as passed-through correctly? I think it's safe to disregard something like the `noIcon` prop we apply to the `Decorator` from the `Pill` because it's not reliant on consumer logic.

I would be more comfortable if we had explicit tests for something like `value` because if for some reason we decide to change the `getByText` query in any of the other assertions, we're no longer ensuring the prop works as expected.

Let me know your thoughts on the above!